src/styles/pages/products/tidb.scss

```diff
@@ -47,6 +75,24 @@
     }
   }
 
+  .comparison-table-section {
```

Comment:
The margin and padding do not come to effect, the `margin-bottom` of `section-title` has been set to `!important` in global.css

Reply:
Please ignore this comment